src/chainstate/stacks/db/mod.rs

```diff
@@ -94,6 +94,11 @@ use core::CHAINSTATE_VERSION;
 
 use chainstate::stacks::db::unconfirmed::UnconfirmedState;
 
+use crate::burnchains::bitcoin::address::BitcoinAddress;
+
+pub static EMPTY_GENESIS_CHAINSTATE_HASH: &str =
```

Comment:
This static should be removed when the hash struct switches to `Sha256Sum` -- if you want an easy way to get the empty value, you should add an `empty()` method to the Sha256Sum impl in `util::hash`